build.xml

```diff
@@ -42,7 +43,7 @@
 
   <target name="compile">
     <mkdir dir="${build}"/>
-    <javac classpathref="build.class.path" debug="on" srcdir="${src}" destdir="${build}" source="1.5" target="1.5"/>
+    <javac classpathref="build.class.path" debug="on" srcdir="${src}" destdir="${build}" source="1.6" target="1.6"/>
```

Comment:
The Installer aims to be compatible with Java 1.5.  Why change this to 1.6?  Does the workflow not support 1.5?

Reply:
Yep, turns out it doesn't.  Will fix.